extern/boostd-data/couchbase/db.go

```diff
@@ -19,6 +19,10 @@ import (
 
 const bucketName = "piecestore"
 
+// maxCasRetries is the number of times to retry an update operation when
+// there is a cas mismatch
+const maxCasRetries = 10
```

Comment:
We should instrument a counter with the CAS retries, so that we know how often this gets hit. If we start getting a lot of CAS retries, we need to revise the optimistic locking, and maybe do pessimistic locks.